l10n_es_mis_report/data/mis_report_pyg_normal.xml

```diff
@@ -0,0 +1,1051 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="mis_report_es_pyg_normal" model="mis.report">
+            <field name="name">Pérdidas y ganancias completo (PGCE 2008)</field>
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_base"
+            />
+        </record>
+        <record id="mis_report_kpi_es_pyg_normal_a" model="mis.report.kpi">
+            <field name="report_id" ref="mis_report_es_pyg_normal" />
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">10</field>
+            <field name="name">esA</field>
+            <field name="description">A) OPERACIONES CONTINUADAS</field>
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l1"
+            />
+            <field name="expression">
+                +es40100 +es40200 +es40300 +es40400 +es40500 +es40600 +es40700 +es40800
+                +es40900 +es41000 +es41100 +es41200 +es41300 +es41400 +es41500 +es41600
+                +es41700 +es41800 +es41900
```

Comment:
Please review this line. In odoo I see this result
![imagen](https://user-images.githubusercontent.com/77978942/113100687-ff3def80-91fb-11eb-9a7d-8e995ffbd7bc.png)
There are some spaces and the balance shows an error